uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/main/java/org/uberfire/ext/metadata/io/BatchIndex.java

```diff
@@ -105,56 +107,69 @@ public void run() {
         }
     }
 
+    private boolean hasContent(Path dir) {
+        // TODO remove this filter when AF-1073 is resolved
+        try (DirectoryStream<Path> children = newDirectoryStream(dir, path -> !path.endsWith("readme.md"))) {
```

Comment:
If this is related with this file https://github.com/kiegroup/appformer/blob/5d491c9afaef4674114b1ed2c2a515c9327a4e54/uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/JGitFileSystemProvider.java#L575

I strong advice to extract both variables and keep them in the same method.

Someday we will change it on FS level and forget about this.

Reply:
Also, what happens if the repo doesn't have a readme.md ? /cc @adrielparedes